todolist.py

```
@@ -55,7 +55,7 @@ def show_incomplete_tasks(self):
 
 # メニューの表示
 def print_menu():
-    print("\n===== Todoリスト =====")
+    prnt("\n===== Todoリスト =====")
```

Ruff (0.8.2)

Undefined name `prnt`

(F821)